src/JavascriptTestTrait.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Drupal\commerce;
```

Comment:
@mglaman Yah I wasn't sure where to put that, after writing a bit more that didn't need javascript, I'm thinking your suggestion above does make the most sense. @bojanz give official blessing?